systemtests/tests/py3plug-fd-postgres/testrunner-default

```diff
@@ -28,6 +28,12 @@ mkdir -p "$TESTPGHOST"
 
 pushd database > /dev/null
 setup_local_db "$TESTPGHOST"
+PSQL_VERSION=$(${PSQL} -qtAX  <<< "SHOW server_version;" | sed 's/\..*//g')
+if [ ${PSQL_VERSION} -ge 15 ]; then
+# skip test
+ echo "${TestName} test skipped: not compatible with PG >= 15"
+ exit 77;
+fi
```

Comment:
I think we can disable the test from within cmake.
```cmake
if(TARGET python3-fd)
  check_pymodule_available(3 pg8000)
  check_pymodule_available(3 dateutil)
  include(FindPostgreSQL)
  if(PYMODULE_3_PG8000_FOUND
     AND PostgreSQL_FOUND
     AND PYMODULE_3_DATEUTIL_FOUND
     AND (${Python3_VERSION_MAJOR} EQUAL 3)
     AND (${PostgreSQL_VERSION_STRING} VERSION_LESS "15.0")
  )
    create_systemtest(${SYSTEMTEST_PREFIX} "py3plug-fd-postgres")
  else()
    create_systemtest(${SYSTEMTEST_PREFIX} "py3plug-fd-postgres" DISABLED)
  endif()
endif()
```
We could similarly disable the postgresql test if we detect pg version < 10.  Not sure what the repercussion of this is though.  What do you think @arogge , @pstorz ?

Reply:
@sebsura I love that solution, as we will in all case build the backup cluster with the installed PostgreSQL version it completely make sense.

Reply:
done